src/common/main.lua

```diff
@@ -324,6 +324,34 @@ function ant.init()
 			Names = msg.Tags.Names,
 		})
 	end)
+
+	Handlers.prepend("_bootAnt", function(msg)
```

Comment:
As discussed, this actually ends up short circuiting the default `_boot` function for the process, which we decided is OK. And as such, to avoid confusion and to signal our overall intent here more clearly, we'll simply update this to:
```
Handlers.once("_boot", function(msg) ... end)
```
which will outright REPLACE the existing boot function and which will run on boot in the exact same way. We can also add a comment indicating that this is our explicit intent.

Reply:
https://github.com/ar-io/ar-io-ant-process/pull/57/commits/7f04c36b536011cec5286215a0ef1bd830c54ee2